crates/snark-acct-types/src/outputs.rs

```diff
@@ -0,0 +1,66 @@
+//! Account output types that get applied to the ledger.
+
+use strata_acct_types::{AcctId, MsgPayload};
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct UpdateOutputs {
+    transfers: Vec<OutputTransfer>,
+    messages: Vec<OutputMessage>,
+}
+
+impl UpdateOutputs {
+    pub fn new(transfers: Vec<OutputTransfer>, messages: Vec<OutputMessage>) -> Self {
+        Self {
+            transfers,
+            messages,
+        }
+    }
+
+    pub fn transfers(&self) -> &[OutputTransfer] {
+        &self.transfers
+    }
+
+    pub fn messages(&self) -> &[OutputMessage] {
+        &self.messages
+    }
+}
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct OutputTransfer {
+    dest: AcctId,
+    value: u64,
+}
```

Comment:
Shouldn't this be a `BitcoinAmount`?

Reply:
Ok you can leave as a TODO or open a ticket? We can discuss refactoring some things out of primitives.